js/src/modals/AddAddress/addAddress.js

```diff
@@ -28,6 +28,7 @@ export default class AddAddress extends Component {
 
   static propTypes = {
     contacts: PropTypes.object.isRequired,
+    address: PropTypes.string,
```

Comment:
So here we separate the `isRequired` props to those optional. I think it's better to be able to see which props should absolutely be passed to a Component easily. Within those two categories, I agree they could be sorted from a-z